### ISSUE TITLE
Picture flips

### ISSUE THREAD

**Issue by Leeeeerich** [denzcoskun/ImageSlideshow#53]
If it getting an image big size, the library picture flips.

**Comment by ChattTheDev** [comment]
Yes, it happened with me too, please tell if you found any solution for that

**Comment by Leeeeerich** [comment]
Hi! Yes, I found some solution. Not best solution, but it problem in Picasso, and team Picasso ignore problems: https://github.com/square/picasso/issues/2201

I get repository, and change Picasso to Glide. And added built to my project.
